src/service/video-manager-impl.js

```diff
@@ -441,6 +509,21 @@ class VideoEntry {
       },
     });
 
+    // Re-measure initial position when the window resizes / orientation changes
+    const viewport = viewportForDoc(this.ampdoc_);
+    viewport.onResize(() => {
+      this.vsync_.run({
+        measure: () => {
+          this.initialRect_ = this.video.element./*OK*/getBoundingClientRect();
```

Comment:
We have `getLayoutBox()` on amp elements (e.g. `this.video.getLayoutBox()` ). Can that be used instead of `initialRect_`? it is cached and updated with resize events. The coordinates are not relative to viewport but looks like you only care about width/height/left anyway?